Bottom: 7ef106cd59875655b2ddf929a941892c87f477e8
Top:    a179935c6eb521104f26ec12dee4c55f0fd2745d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 22:09:43 +0100

Refresh of hid-gtk-drop-in-gl-renderer-wo

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 23b6d46..1745891 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -2,7 +2,7 @@
  *                            COPYRIGHT
  *
  *  PCB, interactive printed circuit board design
- *  Copyright (C) 2011 PCB Contributers (See ChangeLog for details)
+ *  Copyright (C) 2009-2011 PCB Contributers (See ChangeLog for details)
  *
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
@@ -18,9 +18,6 @@
  *  along with this program; if not, write to the Free Software
  *  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  *
- *  Contact addresses for paper mail and Email:
- *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
- *  Thomas.Nau@rz.uni-ulm.de
  */
 
 #ifdef HAVE_CONFIG_H
diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index c26e24e..282b144 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -2,7 +2,7 @@
  *                            COPYRIGHT
  *
  *  PCB, interactive printed circuit board design
- *  Copyright (C) 2009 PCB Contributors (See ChangeLog for details).
+ *  Copyright (C) 2009-2011 PCB Contributors (See ChangeLog for details).
  *
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
